register/index.html

```diff
@@ -0,0 +1,133 @@
+          height="300"
+        />
+      </div>
+
+      <div
+        class="col position-absolute"
+        style="margin-top: -150px; width: 100%"
+      >
+        <img
+          class="m-auto"
+          src="/images/logo-dark.png"
+          alt="logo"
+          width="200"
+          height="200"
+        />
+      </div>
+    </header>
+
+    <main class="container w-75 m-auto p-4">
+      <!--Registration form-->
+      <form
+        action="profile"
+        class="col-xs-12 col-sm-11 col-md-8 col-lg-7 col-xl-6 m-auto was-validated"
+        id="registration-form"
+      >
+        <h1 class="display-3 mb-3 fw-normal text-white">Registration</h1>
+        <div class="form-floating">
+          <input
+            type="text"
+            class="form-control"
+            name="floatingText"
+            id="reg-name"
+            placeholder="First Name Last Name"
+            required
+            pattern="^[\w]+$"
+            title="Username must only contain lowercase and uppercase letters, numbers and underscore (_). Example: My_User123"
+          />
+          <label for="floatingText">Name</label>
+          <div class="valid-feedback">Valid.</div>
+          <div class="invalid-feedback">Please enter your name.</div>
+        </div>
+        <div class="form-floating mt-2">
+          <input
+            type="email"
+            class="form-control"
+            name="floatingInput"
+            id="reg-email"
+            placeholder="name@noroff.no name@stud.noroff.no"
+            required
+            pattern="^[\w\-.]+@(stud.)?noroff.no$"
+            title="Only @(stud.)noroff.no domains are allowed to register"
+          />
+          <label for="floatingInput">Email address</label>
+          <div class="valid-feedback">Valid.</div>
+          <div class="invalid-feedback">Enter a valid email to register.</div>
+        </div>
+        <div class="form-floating mt-2">
+          <input
+            type="password"
+            class="form-control"
+            name="floatingPassword"
+            id="reg-password"
+            placeholder="Password"
+            minlength="8"
+            required
+          />
+          <label for="floatingPassword">Password</label>
+          <div class="valid-feedback">Valid.</div>
+          <div class="invalid-feedback">Password must be 8 characters.</div>
+        </div>
+        <button class="w-100 mt-3 btn btn-lg btn-warning" type="submit">
+          Create Account
+        </button>
+      </form>
+
+      <p class="mt-4 mb-3 text-white">
+        Already have an account? <a href="/" class="text-white">SIGN IN</a>
+      </p>
+      <div class="container">
+        <hr class="text-white w-50 mx-auto mt-5" />
+      </div>
+      <p class="mt-5 mb-3 text-white">
+        By continuing, you agree with our <u>Terms of Service</u>
+      </p>
+    </main>
+    <script src="../node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
```

Comment:
Use the CDN link here if you want to deploy your site to a live server as the node_modules folder does not exist on  production servers, only during development.